extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -273,18 +265,32 @@ public void onConnectionLost(HueBridge bridge) {
      * This method is called whenever the connection to the given {@link HueBridge} is resumed.
      *
      * @param bridge the hue bridge the connection is resumed to
+     * @throws ApiException
+     * @throws IOException
      */
-    public void onConnectionResumed(HueBridge bridge) {
+    private void onConnectionResumed(HueBridge bridge) throws IOException, ApiException {
         logger.debug("Bridge connection resumed. Updating thing status to ONLINE.");
+
+        FullConfig fullConfig = bridge.getFullConfig();
```

Comment:
Is it safe to do this here? It has some smell to me... Note that this is an expensive, potentially long-running and blocking call.